build.sbt

```diff
@@ -268,6 +269,9 @@ lazy val orientdb = alpakkaProject("orientdb",
 lazy val reference = internalProject("reference", Dependencies.Reference)
   .dependsOn(testkit % Test)
 
+lazy val pravega = alpakkaProject("pravega", "pravega", Dependencies.Pravega, fork in Test := true)
+// TODO: revert before merging
```

Comment:
Actually, we'll need to keep this disabled until a release is cut with it.  Update the comment to reference this issue: https://github.com/akka/alpakka/issues/2181